packages/dashboard/src/components/grid/gestures/usePointerTracker.ts

```diff
@@ -39,9 +39,11 @@ export const usePointerTracker = ({ readOnly, enabled, union, click }: PointerTr
   };
 
   const onPointerUp: PointerEventHandler = (e) => {
+    const target = e.target as HTMLDivElement;
     if (cancelClick || !enabled || readOnly) return;
 
-    if (e.button === MouseClick.Left) {
+    /* handle classname 'awsui_modal' to keep focus on selected widget when clicking on modal */
+    if (e.button === MouseClick.Left && !target.ownerDocument.body.className.includes('awsui_modal')) {
```

Comment:
is there any other way to do this. I don't think we should depend on cloudscape classnames since they are not considered stable from their team. Can we use refs instead maybe?